pycuda_cudf_integration/README.md

```diff
@@ -0,0 +1,113 @@
+# Using PyCuda and Cudf
+## Overview
+Sometimes, a user may wish to run custom device code to do numerical operations on a `cuDF` column. 
+
+One way to do this is through custom cython bindings which can require significant setup work as shown [here](https://github.com/rapidsai/rapids-examples/tree/main/shareable-dataframes).
+
+With `PyCUDA`,  the custom CUDA kernels can be run directly using its` sourcModule` to modify the `cuDF`  dataframe.   
+
+`PyCUDA` has limitations, especially around running host-side code, which are noted in detail in the limitations section.  
+
+## PyCUDA Interactions
+In order to write custom cuda kernel in `pycuda` we make use of `SourceModule`. After constructing the kernel, we can retrieve the function and store it in a variable like below.
```

Comment:
Will be good to define what `Source Module` is and add a link it to the relevant documentation.